flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/catalog/PostgresTablePath.java

```diff
@@ -67,6 +67,10 @@ public String getFullPath() {
         return String.format("%s.%s", pgSchemaName, pgTableName);
     }
 
+    public String getQuoteFullPath() {
+        return String.format("\"%s\".\"%s\"", pgSchemaName, pgTableName);
```

Comment:
I would suggest to use _org.apache.flink.connector.jdbc.dialect.JdbcDialect#quoteIdentifier_ which probably should be updated for postgres. 
In this case the solution could be for any jdbc dialect

Reply:
done